test/integration/kubernetes.spec.ts

```diff
@@ -50,8 +64,16 @@ test('deploy sample workloads', async () => {
     kubectl.applyK8sYaml('./test/fixtures/centos-deployment.yaml'),
     kubectl.applyK8sYaml('./test/fixtures/scratch-deployment.yaml'),
     kubectl.applyK8sYaml('./test/fixtures/consul-deployment.yaml'),
-    kubectl.applyK8sYaml('./test/fixtures/cronjob.yaml'),
-    kubectl.applyK8sYaml('./test/fixtures/cronjob-v1beta1.yaml'),
+    kubectl.applyK8sYaml('./test/fixtures/cronjob.yaml').catch((error) => {
+      console.log('CronJob is possibly unsupported', error);
+      cronJobValidator = (_workloads: IWorkloadLocator[]) => true;
+    }),
+    kubectl
+      .applyK8sYaml('./test/fixtures/cronjob-v1beta1.yaml')
+      .catch((error) => {
+        console.log('CronJobV1Beta1 is possibly unsupported', error);
```

Comment:
when I was running the tests I couldn't see the console.logs, where can you see them?

Reply:
should we check if one of these yamls _is_ valid somehow? otherwise both might be invalid?

Reply:
maybe could add the workload count back to below L116 to be > 9 (I think there are 10 total)